packages/runtime/src/runtimes/react/components/Document.tsx

```diff
@@ -14,9 +14,11 @@ export const Document = memo(
     ref: Ref<ElementImperativeHandle>,
   ): JSX.Element {
     return (
-      <DocumentContext.Provider value={document}>
-        <Element ref={ref} element={getRootElement(document)} />
-      </DocumentContext.Provider>
+      <DocumentKeyContext.Provider value={document.key}>
```

Comment:
This is fixing a regression introduced [here](https://github.com/makeswift/makeswift/commit/12b0123e59d403cd36c5b3afd11e262fea9906ca#diff-be45a66a0ddef1e58d09bcd2ab646b3c5fbac719f24d97252d37f3e836a241dfL17).